client/vscode/src/chatActivation.ts

```diff
@@ -14,6 +15,16 @@ export function registerChat(languageClient: LanguageClient, extensionUri: Uri)
 
     panel.webview.onDidReceiveMessage(message => {
         languageClient.info(`vscode client: Received ${JSON.stringify(message)} from chat`)
+
+        // TODO: get server contract types from chat-client
```

Comment:
Those specific contract types should be in `@aws/language-server-runtimes-types` already existing package.

Reply:
Great! I have updated to use the runtime types